Apps/Database/src/Delegates/DBCommunicationDelegate.cs

```diff
@@ -181,5 +216,51 @@ public DBResult<Communication> Delete(Communication communication, bool commit =
             this.logger.LogDebug($"Finished deleting Communication in DB");
             return result;
         }
+
+        private List<Communication> GetActiveBannerCommunications()
+        {
+            List<Communication> communication = this.dbContext.Communication
+                 .OrderByDescending(c => c.CreatedDateTime)
```

Comment:
A list is being returned but only count is being used. Is it necessary to still have an OrderByDescending clause? Order by is an added cost for SQL performance.

Reply:
@BrianMaki  I used OrderByDescending, so I can get the recent expiry date. Incase Nino wants to show the recent effective date in UI